opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -92,7 +92,8 @@ pub trait SpanProcessor: Send + Sync + std::fmt::Debug {
     fn shutdown(&mut self) -> TraceResult<()>;
 }
 
-/// A [`SpanProcessor`] that exports when spans are finished.
+/// A [`SpanProcessor`] that passes finished spans to the configured `SpanExporter`, as
+/// soon as they are finished, without any batching.
```

Comment:
Should it be - 

/// A [`SpanProcessor`] that passes finished spans to the configured `SpanExporter`,
/// without any batching.